config.js

```diff
@@ -140,4 +140,12 @@ config.api.stumbleupon = {
 	accessTokenHeader: config.accessTokenHeader,
 }
 
-
+browser.storage.local.get('clientid', function(obj) {
+	if(obj && obj.clientid) {
+		config.defaultHeaders['X-Su-ClientId'] = obj.clientid.value
+	} else {
+		var guid = Guid.create()
```

Comment:
`Guid.create()` returns a `Guid` object. Probably `Guid.create().value` is the one you're looking for.